### PR TITLE
Np 6775/approval request spid

### PR DESCRIPTION
**Pull request by LenoreChen**
**Please attach to `Release-v3.19`.**
This PR adds `spid` to `ApprovalRequestClassification`

### CODE REVIEW

**Review by jlhughes — COMMENTED** [review]
Assuming we are now adding per spid classifications. We'll need a migration script to update existing classifications for all existing spids that now make use of the previous single classification.   Also, is there logic which enforces the spid on query for classification as I expect all are done via system user/context.

**Review by jlhughes — COMMENTED** [review]
Expecting an nspec setup for the DAO. 
add to foam/nanos/approval/services.jrl

**Comment by LenoreChen** [comment]
> Expecting an nspec setup for the DAO. add to foam/nanos/approval/services.jrl

thanks for the catch, I added to the wrong path (nanopay side)

File: src/foam/nanos/approval/services.jrl [hunk 1]
```diff
@@ -57,3 +57,21 @@ p({
...
   "serviceClass": "foam.nanos.approval.ApprovalRequestNotification"
 })
+
+p({
+  "class":"foam.nanos.boot.NSpec",
+  "name":"releaseCashInApprovalTypeDAO",
+  "serve":true,
+  "serviceScript": """
+    return new foam.dao.EasyDAO.Builder(x)
+      .setJournalType(foam.dao.JournalType.SINGLE_JOURNAL)
```
Comment (jlhughes) [inline src/foam/nanos/approval/services.jrl:68]: Are we manually setting the 'id' property?  If not, add setFuid(true).

**Review by adamfox-nanopay — APPROVED** [review]

**Review by jlhughes — APPROVED** [review]

**Pull request merged**